mysql-test/t/proxy_protocol.test

```diff
@@ -5,3 +5,7 @@ SELECT @@proxy_protocol_networks;
 --error ER_BAD_HOST_ERROR,2013
 connect (err,"127.0.0.1",root,,,$MASTER_MYPORT,);
 --enable_query_log
+
+--echo #PS-5327 proxy protocol-related memory leak at shutdown
+# Restart the server
+--source include/restart_mysqld.inc
```

Comment:
Restart doesn't work with embedded server.

Source `--source include/not_embedded.inc` ad the beginning.